unblob/extractor.py

```diff
@@ -66,15 +66,15 @@ def extract_with_command(
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
         )
-        if res.returncode != 0:
+        if res.returncode == 0:
```

Comment:
Non-0 return code doesn't mean that no output is generated.

Reply:
Right, good catch! The, I will modify the `fix_permissions` to handle when the `path` does not exist.

Reply:
I honestly don't know what mysterious thing triggered the error I created this patch for, but I can not find or reproduce it. Theoretically it can not happen, because:
1. The directory in the `outdir` object is always created here, so at least an empty dir should be there
2. Even if the `outdir` `Path` is invalid, the `Path("<invalid>").rglob("*")` should return with an empty list in the `fix_permissions`, so it would not cause any problems...

So I will simply just drop this commit ... :smiling_face_with_tear: